File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.js
function solution(s, n) {
    let answer = '';
    let bowl = 0;
    for(let i = 0; i < s.length; i++){
        bowl = parseInt(s.charCodeAt(i));
        if(bowl > 64 && bowl < 91){
            bowl += n;
            if(bowl > 90) bowl -= 26;
        }else if(bowl > 96 && bowl < 123){
            bowl += n;
            if(bowl > 122) bowl -= 26;
        }else{
            bowl = 32;
        }
        answer += String.fromCharCode(parseInt(bowl));
    }
    return answer;
}

File: 프로그래머스/lv1/12926. 시저 암호/시저 암호.java
import java.util.*;

class Solution {
    public String solution(String s, int n) {
        String answer = "";
        int bowl = 0;
        for(int i = 0; i < s.length(); i++){
            bowl = (int)s.charAt(i);
            if(bowl >= 97 && bowl <= 122){
                bowl += n;
                if(bowl > 122) bowl -= 26;
            }else if(bowl >= 65 && bowl <= 90){
                bowl += n;
                if(bowl > 90) bowl -= 26;
            }else{
                bowl = 32;
            }
            answer += (char)bowl;
        }
        return answer;
    }
}